tpm/tpm_test.go

```diff
@@ -71,6 +71,22 @@ func TestGetKeys(t *testing.T) {
 	t.Logf("Got %d keys: % d\n", len(handles), handles)
 }
 
+func TestPcrExtend(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+
+	var pcrValue [20]byte
+	var value = "FFFFFFFFFFFFFFFFFFFF"
+	copy(pcrValue[:], value)
+
+	res, err := PcrExtend(rwc, 12, pcrValue)
+	if err != nil {
+		t.Fatal("Couldn't extend PCR 12 from the TPM:", err)
+	}
+
+	t.Logf("Got the new PCR 12 value % x\n", res)
```

Comment:
It'd be nice to check that the value of PCR 12 changed after being extended. Even better would be to check the value to see if it's what we expect (since we know the initial value and the value we're extending it with).